### ISSUE TITLE
Presence of placeholder, redirect or signpost tag on a body to have special effects

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/alaveteli#7347]
The idea as been raised of listing some "bodies" which are not subject to FOI, but not enabling requests to them, just using the entry to point to bodies which are subject to FOI which a request can be made to. 

Examples from WhatDoTheyKnow:

* [Sports Governing Bodies](https://github.com/mysociety/whatdotheyknow-theme/issues/839)
* [Apparent NHS entities that might not really exist as a corporate body](https://github.com/mysociety/whatdotheyknow-theme/issues/1415#issuecomment-1269877578)

The effect of a placeholder or redirect tag should include

* To not be subject to the automatically added missing_email tag feature
* To not show the make a request button

See also:

* https://github.com/mysociety/alaveteli/issues/7298 (also a ticket on [similar] special effects for the forthcoming tag)

**Comment by garethrees** [comment]
`placeholder` seems like it would be more generally useful.

In the case of [`forthcoming`](https://github.com/mysociety/alaveteli/issues/7298) we could use:

* `placeholder` - prevent requests and other special functionality
* `forthcoming` – use this to trigger a tag-based note that "we'll add the body when it exists"

**Comment by mdeuk** [comment]
`redirect` strikes me as perhaps working in a similar way to the [Mediawiki functionality](https://www.mediawiki.org/wiki/Help:Redirects) that some might be familiar with.

A use I could foresee for a feature like that would be where a successor entity takes over - e.g. in the recent case of CCGs in England being replaced by ICBs, it'd have been helpful to render a redirect so that we could automatically point users to the appropriate successor. I _**suspect**_ the most intuitive way of doing this might be to render a tag based note with a link to the successor entity?

**Comment by garethrees** [comment]
> `redirect` strikes me as perhaps working in a similar way to the [Mediawiki functionality](https://www.mediawiki.org/wiki/Help:Redirects)

I'd expect to do like `redirect:an_alternative_body_url_name` which automatically redirects to the alternative body with a short, general note applied to the destination page ("We've redirected you here because X,Y,Z")

I'd expect `placeholder` to limit some functionality, but _not_ automatically take a user to a different URL.

**Comment by garethrees** [comment]
At this point I don't see a strong case for an automated redirect. `placeholder` + notes seems like it would cover the vast majority. of situations

**Comment by RichardTaylor** [comment]
Great examples of where we could use this feature at

https://github.com/mysociety/alaveteli/issues/7369

**Comment by FOIMonkey** [comment]
We're trying this out as a bit of an experiment with Our Future Health 🧑‍🔬 

https://www.whatdotheyknow.com/body/our_future_health

The downside is that missing_email note appears at the mo 😢

**Comment by confirmordeny** [comment]
More examples of why this may be useful ... what would we want people to see if they search for the following?

-  British Government
- Downing Street
- Ministry of Transport
- Secretary of State
- UK Government

**Comment by confirmordeny** [comment]
*** copied from a request that was closed and merged into this one ***

"Add ability to create 'signpost' authority pages that don't allow requests but can point to other authorities. I have suggested something similar to this in the past which was closed but this time I have some specific examples and what I hope is a better reasoning.

My theory is that many people think in terms of buildings, services, brands rather than legal entities. For example, people are more likely to talk about "Thorpe Park" than "Merlin Entertainments Limited". This holds in the public sector also. If I am visiting Chelmsford Library it might not matter which part of local government is running it/funding it. For many the library, leisure centre, museum etc is itself the institution. NHS 111 is a recognised brand.

Examples:
(1) Oban High School is a secondary school in Scotland. We don't list it. I recently discovered from another volunteer that the majority of state schools in Scotland come under the control of the local authority and are not public bodies in their own right in terms of FOISA. You can still request information the school has under FOISA but you need to request it from the local authority. I think this would be far from obvious for most of our current users and potential future users.

(2) We could add individual prisons and then signpost the operator in each case.

(3) Bar Standards Board - might be helpful to list this and point to General Council of the Bar.

(4) Well known law schools, business schools and medical schools that are technically part of larger institutions."

**Comment by RichardTaylor** [comment]
I've just used the signpost tag at 

https://www.whatdotheyknow.com/body/local_government_pension_scheme

**Comment by mdeuk** [comment]
Using a tag based note, and some existing CSS, the _signpost_ tag looks a little better.

The copy is scrappy, so could probably do with some refinement.

<img width="500" alt="Screenshot from WhatDoTheyKnow, showing a message on an orange background which advises readers that they have found a signpost and that they need to refer to a different public body." src="https://user-images.githubusercontent.com/249418/223136361-cc064a5c-9364-4a2b-b5d4-adbba882161e.png">

```html
<div id="request_status" class="request-status-message request-status-message--error_message">
<h3><i class="icon-standalone icon_error_message"></I> Signpost</h3>
We list this entity as a <em>"signpost"</em> to help you find the right authority for your request.
<br />
You can't send a request to <em>this</em> entity from here. If you can't find the body you require, please <a href="/help/contact">contact us</a>, and we'll try to assist.
</div>
```

**Comment by RichardTaylor** [comment]
>There are 18 "signpost" "bodies"/pages now on WhatDoTheyKnow:

Reduced by removing the signpost tag from a set of academies, most of which were defunct. We list academy schools now to help our users, though technically the body subject to FOI is the owner/operator of the academy not the school ittself.

**Comment by RichardTaylor** [comment]
>The effect of a placeholder or redirect tag should include

> *    To not be subject to the automatically added missing_email tag feature
> *    To not show the make a request button

There is a workaround being used on WhatDoTheyKnow:

* Apply the `not_apply` tag to prevent requests being made
* Set the `request_address` to something

**Comment by HelenWDTK** [comment]
Missing email tag being applied to bodies tagged signpost seems like a simple fix. 

Is there some reason that adding

```
  def signpost?
    has_tag?('signpost')
  end
```
to the `public_body` model and changing `update_missing_email_tag` to `if !defunct? && !signpost? && missing_email?` wouldn't work?

**Comment by garethrees** [comment]
This is desirable, but unlikely to be worked on in the next 12 months so closing for now.
